pyproject.toml

```diff
@@ -25,21 +25,22 @@ classifiers = [
 ]
 
 [tool.poetry.dependencies]
-python = ">=3.8,<3.12"
-bokeh = "^2.3"
-numpy = "^1.21"
-pandas = "^1.3"
-pydantic = "^1.8"
+python = ">=3.8"
+bokeh = "^3.0"
```

Comment:
The only suggestion I have is to fix this to 3.4 and keep this updated as the package grows.

Reply:
I've checked with the latest of each 3.* version of Bokeh: 3.1 seems to be the minimum required so I've updated to this.  An fresh install or update to the package should use the latest Bokeh.